CDCgov/NEDSS-Modernization
# [Update] Gradle to 8.4

## Description

Updates the `NEDSS-Modernization` Java projects to use Gradle 8.4.  This allows the containers to be built using the gradle docker image which prevents downloading when building the image.

## Steps to verify

1. Ensure that tests run using `./gradle clean test`
2. Ensure that the `nbs-gateway` container builds using `docker compose up -d --build nbs-gateway`
3. Ensure that the `modernization-api` container builds using `docker compose up -d --build modernization-api`
4. Ensure that the `pagebuilder-api` container builds using `docker compose up -d --build pagebuilder-api`
